server/index.html

```diff
@@ -57,6 +69,7 @@ <h1 class="title"><strong>Hello, Chaos!</strong></h1>
                     </div>
                     <canvas id="canvas"></canvas>
                     <p class="subtitle">Hi, this is <strong>ChaosBot</strong>.</p>
+                    <p class="content">^^^ Hover over me.</p>
```

Comment:
I had no idea there was an animation until I realized there was a JS file. I do agree with you though, there is probably a neater way to alert visitors. :)